src/Fablabsio/FablabsioExtendSocialite.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace SocialiteProviders\Fablabsio;
+
+use SocialiteProviders\Manager\SocialiteWasCalled;
+
+class FablabsioExtendSocialite
+{
+    /**
+     * Register the provider.
+     *
+     * @param \SocialiteProviders\Manager\SocialiteWasCalled $socialiteWasCalled
+     */
+    public function handle(SocialiteWasCalled $socialiteWasCalled)
+    {
+        $socialiteWasCalled->extendSocialite('fablabsio', Provider::class);
```

Comment:
Probably could just remove the `io` from the provider folder and here?
I don't think it conflicts with any other provider?